.github/workflows/test.yml

```diff
@@ -39,22 +39,24 @@ jobs:
           python -m pip install nox
         shell: bash
 
-      - name: Run nox
+      - name: Fix-up PyPy 3.9 executable
```

Comment:
What is this for and why doesn't it affect PyPy 3.7 and 3.8?

Reply:
comment added.